pkg/controller/kabaneroplatform/cli.go

```diff
@@ -83,7 +84,16 @@ func reconcileKabaneroCli(ctx context.Context, k *kabanerov1alpha1.Kabanero, cl
 
 	// Export the github API URL, if it's set.  This is used by the security portion of the microservice.
 	if len(k.Spec.Github.ApiUrl) > 0 {
-		transforms = append(transforms, kabTransforms.AddEnvVariable("github.api.url", k.Spec.Github.ApiUrl))
+		apiUrlString := k.Spec.Github.ApiUrl
+		apiUrl, err := url.Parse(apiUrlString)
+		if len(apiUrl.Scheme) == 0 {
```

Comment:
`url.Parse()` returns a pointer to a `URL`, so I don't think it's safe to call `len(apiUrl.Scheme)` here.  I think we need to check for err first, and then try to parse the scheme.